examples/source/multimedia-animations/Animated_hamburger_menu.html

```diff
@@ -0,0 +1,127 @@
+
+  <!-- Custom styles and CSS animations for the hamburger menu: -->
+    <style amp-custom>
+      .hamburger_wrapper {
+        padding: 5px;
+        z-index: 500;
+      }
+
+      #hamburger {
+        width: 60px;
+        height: 45px;
+        position: relative;
+        transform: rotate(0deg);
+        transition: .5s ease-in-out;
+        cursor: pointer;
+        outline: none;
+
+      }
+      #hamburger span {
+        display: block;
+        position: absolute;
+        height: 9px;
+        width: 100%;
+        background: #0479C4;
+        border-radius: 9px;
+        opacity: 1;
+        left: 0;
+        transform: rotate(0deg);
+        transition: .5s ease-in-out;
+      }
+
+      #hamburger span:nth-child(1) {
+        top: 0px;
+        transform-origin: left center;
+      }
+
+      #hamburger span:nth-child(2) {
+        top: 21px;
+        transform-origin: left center;
+      }
+
+      #hamburger span:nth-child(3) {
+        top: 42px;
+        transform-origin: left center;
+      }
+
+      #hamburger.close span:nth-child(1) {
+        transform: rotate(45deg);
+      }
+
+      #hamburger.close span:nth-child(2) {
+        width: 0%;
+        opacity: 0;
+        transition: .1s;
+      }
+
+      #hamburger.close span:nth-child(3) {
+        transform: rotate(-45deg);
+      }
+      
+      .not-active {
+        position: relative;
+        transform: translateX(-100%);
+        opacity: 0;
+        z-index: 499;
+        transition: transform .5s ease, opacity ease .2s;
+      }
+      .now-active {
+        transform: translateX(0);
+        transition: transform .5s ease, opacity ease .2s;
+        opacity: 1;
+        background-color:bisque;
+      }
+
+      #nav-list {
+        padding: 10px;
+        list-style-type: none;
+        font-size: 2em;
+      }
+    </style>
+</head>
+<body>
+ 
+  <!-- ## Markup -->
+  <!--
+    The hamburger menu element contains three styled `span` elements inside a `div`. The hamburger `div` uses the `tap` AMP action to trigger the `toggleClass()` event, which targets an element by its `id`. In this example, it targets `hamburger`.
+
+    The `toggleClass()` event will add and remove any class it's given as an argument. This example passes `class='close'`. Additionally, it toggles an animation class for the navigation menu to slide out. 
+  -->
+ <div>
+  <div class="hamburger_wrapper">
+    <div id="hamburger" tabindex="1" role="button" on="tap:hamburger.toggleClass(class='close'),nav-menu.toggleClass(class='now-active')">
+      <span></span>
+      <span></span>
+      <span></span>
+    </div>
+  </div>
+  <div id="nav-menu" class="not-active">
+    <ul id="nav-list">
```

Comment:
Is it really necessary to use an id here?
When it is just used for styling we could use `class="nav-list"` and adjust the css above accordingly.